core/templates/dev/head/pages/exploration_editor/settings_tab/SettingsTab.js

```diff
@@ -217,10 +210,7 @@ oppia.controller('SettingsTab', [
         !ExplorationRightsService.viewableIfPrivate());
     };
 
-    /** ******************************************
-    * Methods for notifications muting.
-    ********************************************/
-
+    // Methods for notifications muting.
```

Comment:
```suggestion
    // Methods for muting notifications.
```
Sounds better this way I think.